nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiServiceFacade.java

```diff
@@ -3948,6 +3953,56 @@ public RuntimeManifest getRuntimeManifest() {
         return controllerFacade.getRuntimeManifest();
     }
 
+    @Override
+    public ProcessorDefinition getProcessorDefinition(String group, String artifact, String version, String type) {
+        final ProcessorDefinition processorDefinition = controllerFacade.getProcessorDefinition(group, artifact, version, type);
+        if (processorDefinition == null) {
+            throw new ResourceNotFoundException("Unable to find definition for %s".formatted(type));
```

Comment:
The error message like it should include all of the coordinates instead of just the `type` property.